src/cli.test.ts

```diff
@@ -0,0 +1,292 @@
+import 'source-map-support/register';
+
+import { initArgs, parseArgs } from './cli';
+import { parseJson } from './utils/parseUtils';
+
+describe('initArgs + parseArgs', () => {
+  let mockExit: jest.SpyInstance;
+
+  beforeEach(() => {
+    mockExit = jest.spyOn(process, 'exit').mockImplementation();
+  });
+
+  afterEach(() => {
+    mockExit.mockRestore();
+  });
+
+  test('--help forces exit', () => {
+    // Mock console.log so as not to pullute the log with the yargs help text
+    const mockLog = jest.spyOn(console, 'log').mockImplementation();
+    initArgs(['https://www.google.com', '--help']);
+    expect(mockExit).toHaveBeenCalledTimes(1);
+    expect(mockLog).toBeCalled();
+    mockLog.mockRestore();
```

Comment:
Here and below: why not move this other restore to `afterEach` ?

Reply:
> Here and below: why not move this other restore to `afterEach` ?

In this case, we created the mock for this test only. I didn't want to do it in all tests as that could hide potentially useful output if the test begins failing.

Reply:
Oh sorry, reviewed too fast.